### ISSUE TITLE
Adjust order of elements to account for screen readers / accessibility

### ISSUE THREAD

**Issue by flaneuse** [outbreak-info/outbreak.info#624]
Right now, accessibility readers have trouble with navigating outbreak.info pages, especially when trying to tab navigate through pages.

@mindoftea can help identify particular pages where the order seems out-of-whack.

We should also go through standard accessibility checker to identify other issue -- i know there are many aria tags which are missing, for instance.
